src/components/searchbar.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import { fetchMovies } from '../actions/movies';
+
+
+class Searchbar extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            s: ''
```

Comment:
Even tho parameters in our API aren't self explanatory please keep all variables in our code to be that way.